File: app/src/main/java/mezidia/mezgoodle/quizgame/Stopwatch.java
package mezidia.mezgoodle.quizgame;

import android.annotation.SuppressLint;

public class Stopwatch {
    final int nanoseconds = 1000000000;
    final int seconds = 60;

    public long GetTime() {
        return System.nanoTime() / this.nanoseconds;
    }

    @SuppressLint("DefaultLocale")
    public String GetResult(long time) {
        int minutes = (int) (time / this.seconds);
        int seconds = (int) (time - this.seconds * minutes);
        if (minutes == 0) {
            return String.format("Your time is %d seconds", time);
        } else return String.format("Your time is %d minute(s) and %d seconds.", minutes, seconds);
    }
}

File: app/src/main/java/mezidia/mezgoodle/quizgame/Stopwatch.kt
package mezidia.mezgoodle.quizgame

import android.annotation.SuppressLint

class Stopwatch {
    private val nanoseconds = 1000000000
    private val seconds = 60

    // Get current time
    fun getTime(): Long {
        return System.nanoTime() / nanoseconds
    }

    // Get final time
    @SuppressLint("DefaultLocale")
    fun getResult(time: Long): String {
        val minutes = (time / seconds).toInt()
        val seconds = (time - seconds * minutes).toInt()
        return if (minutes == 0) {
            String.format("Your time is %d seconds", time)
        } else String.format("Your time is %d minute(s) and %d seconds.", minutes, seconds)
    }
}